<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_kesilen">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KESİLEN KARPUZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_kacan">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>50</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KAÇAN KARPUZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sayac">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_arka">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1920</width>
      <height>980</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resimler/images/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
